site/_posts/2016-06-03-update-on-jekyll-s-google-summer-of-code-projects.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: news_item
+title: "Jekyll's Google Summer of Code Project: The CMS You Always Wanted"
+date: "2016-06-03 13:21:02 -0700"
+author: parkr
+categories: [community]
+---
+
+This year, Jekyll applied to be a part of [Google Summer of Code](https://summerofcode.withgoogle.com/how-it-works/). Students were able to propose any project related to Jekyll. With a gracious sponsorship from GitHub and the participation of myself, [@benbalter](https://github.com/benbalter) and [@jldec](https://github.com/jldec), Jekyll was able to accept two students for the 2016 season, [@mertkahyaoglu](https://github.com/mertkahyaoglu) and [@rush-skills](https://github.com/rush-skills).
+
+These students are working on a project that fills a huge need for the community: **a graphical solution for managing your site's content.** Current plans include a fully-integrated admin which spins up when you run `jekyll serve` and a web interface at an address like `http://localhost:4000/admin/`. The server implements a common interface which would make a hosted version to make updates to hosted content like a repository on GitHub very easy to write – simply implement the CRUD API and the web interface will happily use that instead.
```

Comment:
@benbalter I wanted to be generic because it could morph into anything. But the key being a friendlier user interface than a text editor. 😄 Hm... Like:

> Current plans include a fully-integrated admin which spins up when you run `jekyll serve` and provides a friendly web interface for creating and editing your content.